src/Native/src/ntt/cpu_runtime.cmake

```diff
@@ -29,11 +29,10 @@ else()
     endif()
 endif()
 
-add_library(nncase_cpu_runtime STATIC ${CMAKE_CURRENT_LIST_DIR}/cpu_runtime.cpp)
-target_compile_features(nncase_cpu_runtime PUBLIC cxx_std_20)
-target_include_directories(nncase_cpu_runtime PUBLIC ${CMAKE_CURRENT_LIST_DIR}/../include)
-
-if (MSVC)
-    set_property(TARGET nncase_cpu_runtime PROPERTY
-        MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
-endif()
```

Comment:
What's the reason of deleting these lines?

Reply:
To address the multi-kernel symbol conflict problem, I removed the “nncase_cpu_runtime” and moved this source code into [Main.cpp.cshtml](https://github.com/kendryte/nncase/pull/1185/files#diff-10fa901060d9e28abca04f58bbc2fddbe42f150245d522d5bc7ae79e268329f7)

Reply:
Multi kernel should be linked into one elf.

Reply:
restored the modification, Next PR will add the link of the module‘s kernels.